src/audio_core/hle/wmf_decoder.cpp

```diff
@@ -163,9 +163,10 @@ MFOutputState WMFDecoder::Impl::DecodingLoop(ADTSData adts_header,
             }
         }
 
-        // in case of "ok" only, just return quickly
+        // xperia64: If we return OK here, the decoder won't be in a state to receive new data and
```

Comment:
```suggestion
        // If we return OK here, the decoder won't be in a state to receive new data and
```

The name in particular can be omitted, since it's not particularly relevant here (plus, a blame on the code will direct back to the commit anyways, which has your authored name associated to it).